master/buildbot/schedulers/basic.py

```diff
@@ -36,7 +36,7 @@ class BaseBasicScheduler(base.BaseScheduler):
     """
 
     compare_attrs = ['treeStableTimer', 'change_filter', 'fileIsImportant',
-                     'onlyImportant', 'reason']
+                     'onlyImportant', 'reason', 'coolDownTimer']
```

Comment:
Side note: I think it'd be the best if we had all `compare_attrs` formatted with one item per line and sorted by name...